frontend/appflowy_tauri/src/appflowy_app/components/_shared/EditRow/EditCellDate.tsx

```diff
@@ -17,7 +17,7 @@ export const EditCellDate = ({
   };
 
   return (
-    <div ref={ref} onClick={() => onClick()} className={'w-full px-4 py-2'}>
+    <div ref={ref} onClick={() => onClick()} className={'w-full px-4 py-1'}>
```

Comment:
Maybe I remember incorrectly, but isn't it possible to just define onClick like `onClick={onClick}`

Reply:
these onClick functions are not same in this particular situation, I'll rename the function to make it less confusing.

Reply:
yep, agreed